docs/snakefiles/modularization.rst

```diff
@@ -301,6 +301,12 @@ For the latter, it is also possible to specify an alternative host, e.g.
 
     gitlab("owner/repo", path="workflow/Snakefile", tag="v1.0.0", host="somecustomgitlab.org")
 
+A private token can be specified to access private repositories:
+
+.. code-block:: python
+
+    github("owner/repo", path="workflow/Snakefile", tag="v1.0.0", token="xxx_ccluNyxdHXXXXX9H6nvC873n3lLG")
+    gitlab("owner/repo", path="workflow/Snakefile", tag="v1.0.0", host="somecustomgitlab.org", token="xxxxx-1fhwBFYCDh-QbB")
```

Comment:
```suggestion
    gitlab("owner/repo", path="workflow/Snakefile", tag="v1.0.0", host="somecustomgitlab.org", token="xxxxx-1fhwBFYCDh-QbB")
    
Naturally, in reality you should query these tokens from environment variables (using ``os.environ``) instead of hardcoding them into the source file.
```